blaze/server/server.py

```diff
@@ -219,7 +209,7 @@ def run(self, port=DEFAULT_PORT, retry=False, **kwargs):
 
 
 @api.route('/datashape', methods=['GET'])
-@crossdomain(origin='*', methods=['GET'])
+@cross_origin(origins='*', methods=['GET'])
```

Comment:
We need to consider if we want to let any and all browser content to load from the Blaze server, as the `*` essentially defeats the cross-origin content loading protection. Might want to eventually move toward making this configurable or default to localhost:6363?